### DISCUSSION TITLE
Customization Level Supported For Side Controls

### DISCUSSION THREAD

**Discussion by davidbrinton** [optimajet/formengine#29] [Q&A] [answered]
We have a requirement to make the panel showing on the left of the Form Engine a detached slide-out panel. We'll hide the top navigation for going to the Tree View or Settings and instead add some tabs where we can show or hide different segments of the available components. Is this something that's supported? I can't find documentation on customization to this degree. If there is some can you point me in the right direction?

![image](https://github.com/user-attachments/assets/b23954f1-a1fb-4251-9015-04d6deb83cdc)

Thank you

**Comment by sergeythrees** [comment, marked as answer]
Hi, @davidbrinton 
[Here ](https://formengine.io/documentation/designer-customization#complete-list-of-customizable-component-keys) is the documentation section regarding customization.
The current customization options are very flexible and allow you to replace almost any part of the interface. But if I understood correctly, you want to place categories in tabs instead of headers, i.e. change the way the list of components is displayed.  At the moment, this is only possible when editing the source code.  
If you purchase the [paid version](https://formengine.io/pricing/), we can extend the customization functionality for you so that this change is possible without editing the source code.
We also have a source code version available, in case you need deeper customization.

**Reply by davidbrinton** [reply]
@sergeythrees Thank you, we have purchased the paid version. Would love to take advantage of that feature if and when it gets added.

**Reply by sergeythrees** [reply]
> @sergeythrees Thank you, we have purchased the paid version. Would love to take advantage of that feature if and when it gets added.

Please write to our support so that we can discuss the improvements in more detail. Also send us the email address to which you made the purchase.

https://formengine.io/contacts
